lib/tasks/sidekiq/cleanup.rake

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'three_scale/sidekiq_batch_cleanup_service'
+
+namespace :sidekiq do
+  desc 'cleanup BID-* keys from sidekiq-batch, specify the max age in seconds as an argument'
+  task :cleanup_batches, [:max_age_seconds] => :environment do |task, args|
+    max_age_seconds = args[:max_age_seconds].to_i
+
+    Rails.logger.info "Cleaning up BID-* keys older than #{max_age_seconds.seconds.in_hours} hours from sidekiq-batch..."
+
+    ThreeScale::SidekiqBatchCleanupService.call(max_age_seconds: max_age_seconds)
```

Comment:
```suggestion
    ThreeScale::SidekiqBatchCleanupService.call(max_age_seconds:)
```

In ruby 3.1 we can skip passing argument if its name is same as the named argument.

Reply:
Oh yes, that's true, thanks!
Indeed, there are some nice improvements in 3.1 https://rubyreferences.github.io/rubychanges/3.1.html

I'll keep it as it is for the time being, as I want to run this in production, where 3.1 is not available yet, but then I will update.

Another interesting thing is that when passing a block as an argument, we can skip the name and just pass `&`.